### DISCUSSION TITLE
Create an auto `man` page

### DISCUSSION THREAD

**Discussion by tjapro** [tldr-pages/tldr#10214] [Ideas]
I know this seems to be irrelevant and/or redundant, but sometimes we need more info about `tldr`. In this case, would be better to have more information in `man tldr` or `man tealdeer`.  
Currently, there is no man page.

In the `man` page could be all auto-parsed documentation from [github pages](https://dbrgn.github.io/tealdeer/).

Any thoughts?

**Comment by kbdharun** [comment]
We do have an auto man page creation action on releases for our C client and also docs (man page) for Python client, the only official clients missing it is Node, extldr from what I remember.

Also, tealdeer is a third party client so I would suggest creating the same issue there, since it's written in rust maybe man page generation can be done easily using rustdoc.

**Reply by tjapro** [reply]
Yup, thanks @kbdharun for your answer. I'll move this discussion to the `tealdeer` repo, because this is a client issue (not a `tldr` issue).

**Comment by tjapro** [comment]
New issue: https://github.com/dbrgn/tealdeer/issues/325

Closing this discussion.
